zadaca5/src/zadatak1/Performance.java

```diff
@@ -0,0 +1,71 @@
+package zadatak1;
+
+import java.util.Scanner;
+
+public class Performance {
+    private int KPIScore;
+    private double coefficient;
+    private double brutoSalary;
+
+    public Performance () {
+        this.enterValues();
+    }
+    public Performance (int KPIScore, double brutoSalary) {
+        this(KPIScore,brutoSalary,0.5);
+    }
+    public Performance (int KPIScore, double brutoSalary, double coefficient) {
+        this.KPIScore = KPIScore;
+        this.brutoSalary =brutoSalary;
+        this.coefficient = coefficient;
+    }
+
+    public double getBrutoSalary() {
+        return brutoSalary;
+    }
+
+    public double getCoefficient() {
+        return coefficient;
+    }
+
+    public int getKPIScore() {
+        return KPIScore;
+    }
+
+    public void setBrutoSalary(double brutoSalary) {
+        this.brutoSalary = brutoSalary;
+    }
+
+    public void setCoefficient(double coefficient) {
+        this.coefficient = coefficient;
+    }
+
+    public void setKPIScore(int KPIScore) {
+        this.KPIScore = KPIScore;
+    }
+
+    public double bonusAmount(){
+        if (this.KPIScore >= 50){
+        return this.coefficient*this.KPIScore*this.brutoSalary/100;}
+        else return 0;
+    }
+
+    public void meetExpectations(){
+        if (this.KPIScore==100){
+            System.out.println("Congratulations, you have met the expectations, you will be rewarded with bounus amount of: " + this.bonusAmount());
+        }else if (this.KPIScore>100) {
+            System.out.println("Congratulations, you have exceeded the expectations, you will be rewarded with bounus amount of: " + this.bonusAmount());
+        } else if (this.KPIScore<100 && this.KPIScore>50) {
+            System.out.println("Unfortunately, you didn't met the expectations, but you haven't either underperformed. You will receive a bonus in amount of: " + this.bonusAmount()/2);
+        }else if (this.KPIScore<=50) {
+            System.out.println("Unfortunately, you have underperformed.");
+        }
+    }
```

Comment:
I've seen this a lot in other homework.

If the method is called `meetExpectations()` I as user of this class would expect the method to return a `boolean` or something, but not to print stuff out.

In your case true/false is not enough (you can make the logic simpler to fit this), so in such case we would usually use an `Enum`, but this goes a bit out of the scope currently, so i suggest to make the logic easier.. https://www.w3schools.com/java/java_enums.asp

The key takeaway is, printing should be done only by a `printMethod()`, other methods should not print, but calculate some value and return it, or mutate the state of the object in fashion which fits the methoName i.e. : `reduceKPI(final double value)` would probably reduce the KPI by `value`.

Also formatting is horrific.

Reply:
Yes, I noted that it was suggesting even in IntelliJ, but didn't know the logic. Will adjust it :)

Reply:
Noted. I did notice that even IntelliJ was suggesting that it should return Boolean, but I wasn't aware of the logic behind it. Thanks for heads up, I will adjust it.